docs/howto/operator/bash.rst

```diff
@@ -41,6 +41,37 @@ You can use :ref:`Jinja templates <jinja-templating>` to parameterize the
     :start-after: [START howto_operator_bash_template]
     :end-before: [END howto_operator_bash_template]
 
+
+.. warning::
+
+    Care should be taken with "user" input or when using Jinja templates in the
+    ``bash_command``, as this bash operator does not perform any escaping or
+    sanitization of the command.
+
+    This applies mostly to using "dag_run" conf, as that can be submitted via
+    users in the Web UI. Most of the default template variables are not at
+    risk.
+
+For example, do **not** do this:
+
+.. code-block:: python
+
+    bash_task = BashOperator(
+        task_id="bash_task",
+        bash_command='echo "Here is the message: \'{{ dag_run.conf["message"] if dag_run else "" }}\'"',
+    )
+
+Instead, you should pass this via the ``env`` kwarg and use double-quotes
+inside the bash_command, as below:
+
+.. code-block:: python
+
+    bash_task = BashOperator(
+        task_id="bash_task",
+        bash_command='echo "here is the message: \'$message\'"',
```

Comment:
This lin failed again, I mean you can remove the quotes after echo as it is just bash.

Example:

```suggestion
        bash_command='echo here is the message: '$message"',
```

Reply:
I guess I can. The point was to try and get output when run like

> here is the message 'foo bar'

Reply:
Running it locally, the problem does appear to be around `\`

![image](https://user-images.githubusercontent.com/34150/83866891-68f68580-a720-11ea-80ad-1091f80423d9.png)

Reply:
Oh pay attention ash. It's not this file.

Reply:
https://github.com/apache/airflow/pull/9143/commits/c8fd7de369dff5fcf9b5950b034330292f78062b